tardis/montecarlo/tests/test_montecarlo.py

```diff
@@ -96,41 +96,6 @@ def fin():
     return compare_data
 
 
-@pytest.fixture(scope="function")
```

Comment:
This and ion_edges() are both continuum tests too

Reply:
Added them back